mps_youtube/players/generic_player.py

```diff
@@ -0,0 +1,57 @@
+import os
+import subprocess
+
+from .. import config
+
+from ..player import CmdPlayer
+
+#
+# This class can be used as a templete for new players
+#
+# NOTE:
+# If you're defining a new player donot forget
+# to name both the class and file the same as your player
+#
+
+
+class generic_player(CmdPlayer):
```

Comment:
Classes in Python conventionally have CamelCase names (see [PEP 8](https://www.python.org/dev/peps/pep-0008/#class-names)). Is there any reason not to follow that here?

Reply:
Sorry, I'm quite new to open source and good coding practices.